code/game/world.dm

```diff
@@ -415,6 +415,29 @@ GLOBAL_VAR(restart_counter)
 
 /world/proc/refresh_atmos_grid()
 
+/world/proc/change_fps(new_value = 20)
+	if(new_value <= 0)
+		CRASH("change_fps() called with [new_value] new_value.")
+	if(fps == new_value)
+		return //No change required.
+
+	fps = new_value
+	on_tickrate_change()
+
+
+/world/proc/change_tick_lag(new_value = 0.5)
```

Comment:
is this even used anywhere?

Reply:
Nope, nothing calls it. Not even tg uses it.

Reply:
ill just comment it for now. aec22d0

Reply:
I mean is tick_lag set anywhere else

Reply:
No, not from what I can see